nomad/grapl-core.nomad

```diff
@@ -0,0 +1,428 @@
+# For rust logging output
+variable "RUST_LOG" {
```

Comment:
The fact that this is the only variable in SCREAMING_SNAKE_CASE is odd... we should name them all consistently.

Reply:
True! Fixed :)

I also realized I never used the variable! I think it was the very first thing I wrote in the job and forgot to use it lol